pkg/jobrunaggregator/jobrunhistoricaldataanalyzer/types.go

```diff
@@ -0,0 +1,29 @@
+package jobrunhistoricaldataanalyzer
+
+import (
+	"time"
+
+	_ "embed"
+
+	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorapi"
+)
+
+//go:embed pr_message_template.md
+var prTemplate string
+
+type parsedJobData struct {
+	NoPrevData                            bool          `json:"-"`
+	TimeDiff                              time.Duration `json:"-"`
+	DurationP95                           time.Duration `json:"-"`
+	DurationP99                           time.Duration `json:"-"`
+	jobrunaggregatorapi.HistoricalDataRow `json:",inline"`
+}
+
+type compareResults struct {
+	increaseCount   int
+	decreaseCount   int
+	missingDataList string
```

Comment:
What do we need this for?

Reply:
Good catch, that's a left over from changing it to be an array of the jobs, removing it.